### PR TITLE
OP-527 inconsistent column definition for XXX_DELETED fields

### PR DESCRIPTION
**Pull request by mwithi**
See OP-527.

### CODE REVIEW

File: src/main/java/org/isf/admission/model/Admission.java [hunk 1]
```diff
@@ -40,6 +40,7 @@
 import javax.persistence.Version;
 import javax.validation.constraints.NotNull;
 
+import org.hibernate.annotations.ColumnDefault;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/admission/model/Admission.java:43]: This import is not used.

File: src/main/java/org/isf/admission/model/Admission.java [hunk 2]
```diff
@@ -196,8 +197,8 @@ public class Admission extends Auditable<String> implements Comparable<Admission
 	private int lock;                        // default 0
 
 	@NotNull
-	@Column(name = "ADM_DELETED")
-	private String deleted;                    // flag record deleted ; values are 'Y' OR 'N' default is 'N'
+	@Column(name = "ADM_DELETED", columnDefinition = "char(1) default 'N'")
+	private Character deleted = 'N';                // flag record deleted ; values are 'Y' OR 'N' default is 'N'
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/admission/model/Admission.java:201]:
```suggestion
	private char deleted = 'N';                // flag record deleted ; values are 'Y' OR 'N' default is 'N'
```

This way there is no boxing/unboxing of the field.    All unit tests pass with this change.

File: src/main/java/org/isf/patient/model/Patient.java [hunk 3]
```diff
@@ -174,8 +174,8 @@ public class Patient extends Auditable<String> {
 	private String profession;
 
 	@NotNull
-	@Column(name="PAT_DELETED")
-	private String deleted = "N";
+	@Column(name="PAT_DELETED", columnDefinition = "char(1) default 'N'")
+	private Character deleted = 'N';
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/patient/model/Patient.java:178]:
```suggestion
	private char deleted = 'N';
```

Same change as noted above.

**Review by dbmalkovsky — CHANGES_REQUESTED** [review]
Please see suggested changes.   Otherwise LGTM.

File: src/main/java/org/isf/admission/model/Admission.java (hunk 1 above)
Comment (mwithi) [reply src/main/java/org/isf/admission/model/Admission.java:43]: Removed, thanks!

**Pull request merged**